.changeset/plenty-jars-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+Fixed the size of the `Input` component's optional `prefix` and `suffix` (when passed as strings). They are now 16px (the size of the input's placeholder and value) instead of 14px (the size of the input's label).
```

Comment:
I looked into this a bit more because it was confusing me. Including padding, the `prefix` and `suffix` measure 48px by 48px. The content is 16px by 24px (this is the change you're referring to, right?). Strings overflow if they exceed 16px.

I wonder why the content isn't square like its container. Should we change that?

Reply:
Sorry for the unclarity here. The change is the font size, not the padding:

| Before | After |
| --- | --- |
| <img width="324" alt="Screenshot 2022-10-19 at 10 39 55" src="https://user-images.githubusercontent.com/35560568/196641218-e6b3e471-dfbe-463d-a246-3624420a8d9f.png"> | <img width="324" alt="Screenshot 2022-10-19 at 10 40 06" src="https://user-images.githubusercontent.com/35560568/196641340-51841cf1-ae04-49b3-b1d9-53c2530b54be.png"> | 

(subtle difference, I know, but now aligned with Figma specs)

Changing the padding to make the content area square wouldn't make a very big difference except slightly shift the prefix/suffix towards the centre (since text is centred in the wrapper).

Figma specs are actually inconsistent: with a currency as a suffix, the suffix's padding is 12px on all sides. With an icon, it's 12px vertically and 16px horizontally (matches the current implementation). A challenge is that the prefix/suffix content is an unknown width but needs to be absolutely positioned since it's out of the input (this could also be rethought).

I'm tempted to release this as-is to avoid even more changes and having to loop designers in. Thoughts?

Reply:
Works for me :)